pkg/chart/loader/local.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright The Helm Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package loader
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// ExpandLocalPath expands a local file, dir or glob path to a list of files
+func ExpandLocalPath(path string) ([]string, error) {
```

Comment:
`Local` should be `File` - "local" is a little too generic in this context. A local URL path like `http://localhost/my-chart.tgz` is still considered by many to be a "local" path.

Reply:
> Hello, Somebody has any update about this? Like @anamarija said, we're using Argo CD too, and it's a common approach getting the configuration from another repository. Thanks a lot!

You're right. I also really want this feature released!
I'm sorry for delaying this for so long. I went on a trip after working on this, and then when I came back I couldn't find the time to continue this. I'll try to put some effort on this in the next days so we can get this done!
You're welcome to join me :)